### DISCUSSION TITLE
`System.Diagnostics.Debug` output in WASM

### DISCUSSION THREAD

**Discussion by MartinZikmund** [unoplatform/uno#14298] [Q&A]
When debugging a WASM app, the `Debug.Write` output is not being redirected to the debug output window nor the Developer tools. `Console` output is displayed in Developer tools only. Is this a Uno-specific limitation or a .NET WASM limitation?

**Comment by jeromelaban** [comment]
This is a runtime issue, indeed. I don't see a reason for this not working, but it's likely possible to fix this in the runtime.
